capstone/capdb/migrations/0049_auto_20180820_1834.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 2.0.8 on 2018-08-20 18:34
+
+import django.contrib.postgres.indexes
+import django.contrib.postgres.search
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('capdb', '0048_merge_20180820_1827'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='casemetadata',
+            name='tsvector',
+            field=django.contrib.postgres.search.SearchVectorField(blank=True, null=True),
```

Comment:
Maybe worth deleting and recreating the migrations to avoid this alterfield? I'm not sure how long it will take to alter field on casemetadata.

Reply:
Good call.